mlir/lib/Dialect/QC/Translation/qasm3/QASM3Emitter.cpp

```
@@ -0,0 +1,1477 @@
+    if (initializer != nullptr) {
+      return assignFloat(id, *initializer);
+    }
+    return success();
+  }
+
+  LogicalResult boolAssign(SMLoc /*loc*/, StringRef id,
+                           const Condition& value) {
+    return assignBool(id, value);
+  }
+
+  LogicalResult intAssign(SMLoc /*loc*/, StringRef id, const Expr& value) {
```

Cppcheck (2.21.0)

[style] The function 'intAssign' is never used.

(unusedFunction)